python/extractor/tsg-python/Cargo.toml

```diff
@@ -6,7 +6,7 @@ version = "0.1.0"
 authors = ["Taus Brock-Nannestad <tausbn@github.com>"]
 edition = "2018"
 
-# When changing/updating these, the `Cargo.Bazel.lock` file has to be regenerated.
+#  When changing/updating these, the `Cargo.Bazel.lock` file has to be regenerated.
```

Comment:
Was this whitespace added to change the hash of this file? If so, I would prefer something a bit more explicit like what we do here for the autogenerated Python `dbscheme`: https://github.com/github/codeql/blob/main/python/extractor/semmle/dbscheme.template#L1

Reply:
It's just a `Dummy change to trigger CI`; I actually have a meaningful change I'll make to the file instead.

Reply:
Ah, okay. In that case, carry on. 🙂